runtime/aardvarks/info.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apigeeregistry/v1
+items:
+  - apiVersion: apigeeregistry/v1
+    kind: API
+    metadata:
+      name: apigee-apihub-demo-fauna-aardvarks-proxy
+      labels:
+        apihub-business-unit: apigee-apihub-demo
+        apihub-kind: proxy
+        source: generate-animal-apis
+      annotations:
+        apigee-proxy: apigee-apihub-demo/apis/fauna-aardvarks
+    data:
+      displayName: 'apigee-apihub-demo proxy: fauna-aardvarks'
+      recommendedVersion: v1
+      deployments:
+        - metadata:
+            name: bar-org
+            labels:
+              apihub-gateway: apihub-google-cloud-apigee
+            annotations:
+              apigee-envgroup: organizations/apigee-apihub-demo/envgroups/bar
```

Comment:
How about we add an annotation for "apigee-organization" and drop the organization from all these other apigee-* annotations.

Reply:
Speaking of annotations, we don't allow users to query or filter based on annotations... that seems like it's likely to be an issue - would customers not want to be able to search on these fields?

Reply:
I agree with the "apigee-organization" suggestion -- that seems like good data normalization.

We could also move these to labels, but only if they don't violate the character restrictions on labels. I think that's hard to guarantee, so we might have to address the filtering problem by changing something else, like bending (extending?) the API guidelines to also filtering on annotations.

Reply:
I think the signal source author will likely just need to determine what ought to be directly searchable data and put that in labels. But we should also keep in mind that this data won't generally be directly searched, but will be exported in order to be processed and linked to enrolled APIs.

Reply:
These annotations are normalized and flattened to just contain ids (the final path segment):
```
            annotations:
              envgroup: demo
              environment: test-env
              message-count-7-days: "2"
              organization: apigee-apihub-demo
              proxy: petstore
              proxy-revision: "1"
```